projects/fsastorefrontlib/src/cms-components/agent/agent-search-list/agent-search-list.component.html

```diff
@@ -37,6 +37,35 @@
           </button>
         </div>
       </ng-container>
+      <div
+        class="col-lg-6 col-12 mb-5"
+        [class.md-order-change]="singleAgentSelected"
```

Comment:
Added `singleAgentSelected` property to control map position on mobile more easily.